.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Pull Request
+title: ""
+assignees: ""
+---
+
+<!-- Please Please follow the official guide from the OCA Guidelines page (https://odoo-community.org/page/contributing)  -->
```

Comment:
I suggest adding something link "please provide a summary of what your PR does and why it is necessary or useful."